<launch>
  <node name="image_view" pkg="image_view" type="image_view" output="screen" >
  <remap from="/image" to="/usb_cam/image_raw"/>
  </node>

  <include file="$(find lab4_cam)/launch/run_cam.launch" />
  <include file="$(find lab4_cam)/launch/ar_track.launch" />

  

</launch>
